src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/reranker/impl/AmazonPersonalizedRankerImpl.java

```diff
@@ -10,11 +10,13 @@
 import com.amazonaws.services.personalizeruntime.model.GetPersonalizedRankingRequest;
 import com.amazonaws.services.personalizeruntime.model.GetPersonalizedRankingResult;
 import com.amazonaws.services.personalizeruntime.model.PredictedItem;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Using `StringUtils` requires adding the Apache Commons Lang library as a dependency. While this is not a significant issue, it does introduce an additional external dependency that needs to be managed and potentially increases the project's size. Also, calling utility methods for simple string operations (only isEmpty is used) can add some overhead compared to inline operations.

Reply:
Sure. Will replace it with ``!=null ``and ``isEmpty()`` checks on the ``String`` object itself.